vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow-integration-tests/src/main/java/com/vaadin/flow/component/richtexteditor/tests/RichTextEditorI18nPage.java

```diff
@@ -0,0 +1,30 @@
+package com.vaadin.flow.component.richtexteditor.tests;
+
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.richtexteditor.RichTextEditor;
+import com.vaadin.flow.router.Route;
+
+@Route("vaadin-rich-text-editor/i18n")
+public class RichTextEditorI18nPage extends Div {
+    public RichTextEditorI18nPage() {
+        RichTextEditor editor = new RichTextEditor();
+        editor.setI18n(createCustomI18n());
+
+        Div i18nOutput = new Div();
+        i18nOutput.setId("i18n-output");
+        i18nOutput.setText(editor.getI18n().toString());
```

Comment:
This stores the I18N config from the server on the page, the IT then takes that and compares it to the web components `i18n` property.